presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -2782,4 +2781,16 @@ public static SchemaTableName getSourceTableNameForPartitionsTable(SchemaTableNa
                 tableName.getSchemaName(),
                 tableName.getTableName().substring(0, tableName.getTableName().length() - PARTITIONS_TABLE_SUFFIX.length()));
     }
+
+    private static ConnectorTableLayout makeEmptyTableLayout()
```

Comment:
nit: consider inlining for ease of reading (to not require the reader to jump to makeEmptyTableLayout to understand what it is)